com.unity.render-pipelines.universal/Shaders/PostProcessing/GaussianDepthOfField.shader

```diff
@@ -1,13 +1,9 @@
 Shader "Hidden/Universal Render Pipeline/GaussianDepthOfField"
 {
-    Properties
-    {
-        _MainTex("Source", 2D) = "white" {}
-    }
-
     HLSLINCLUDE
 
         #pragma target 3.5
+        #pragma multi_compile _ _USE_DRAW_PROCEDURAL
```

Comment:
Same here: adding more "multi_compile _ " will only increase shader build times. For fill-rate intensive postprocessing, use quad mesh is sufficient.